Dockerfile

```diff
@@ -1,13 +1,23 @@
-FROM node:12.18.1-alpine
-RUN apk add --update\
- python \
- python3 \
- build-base \
- zeromq-dev \
- && rm -rf /var/cache/apk/*
+FROM ubuntu:20.04
+
+ENV NODE_VERSION=12.18.1
+ENV NVM_DIR=/usr/local/.nvm
```

Comment:
i don't _mind_ NVM (and it's great for development), but not sure if you need it since this will only ever need exactly one node version

Reply:
Please see the comment below, it was intentional to use a particular(12.18.1) version.